tests/idea-space.spec.ts

```diff
@@ -41,6 +41,21 @@ test.describe('/api/idea-cards', () => {
     });
 });
 
+test.describe('/api/comments', () => {
+
+    test('should add comment', async ({ request }) => {
+        const newComment = await api(request).post("/api/comments", {
+            idea_card: { id: ideaId },
+            text: "test comment"
+        });
+        commentId = newComment.id;
+        const comment = await api(request).getData(`/api/comments/${commentId}`);
+        expect(comment.attributes.text).toBe("test comment");
+    })
+
+});
+
+/*
```

Comment:
@oriyomibadmus we are safe to delete the test case below since it contradicts your new test above